snowflake/{{ cookiecutter.repo_name }}/src/requirements.txt

```diff
@@ -0,0 +1,13 @@
+black~=22.0
+flake8>=3.7.9, <4.0
+ipython>=7.31.1, <8.0
+isort~=5.0
+jupyter~=1.0
+jupyterlab~=3.0
+kedro~={{ cookiecutter.kedro_version }}
+kedro-datasets[spark.SparkDataSet]
```

Comment:
Do we need this for snowpark starter? Or shall we add/replace with SnowParkDataSet?

Reply:
Good catch, should be SnowParkDataSet